### ISSUE TITLE
BUG - sameSite=none cookie warning in firefox

### ISSUE THREAD

**Issue by tjanowsk** [Furlough-Friends/rota-flex-101#138]
![cookie_warning](https://user-images.githubusercontent.com/63243951/82449235-cfce3a80-9aa2-11ea-8cc0-fbaa85771c83.png)

Website it points to:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

**Comment by emab** [comment]
I don't think this will be a problem. Auth0 have addressed this issue, and in our case we won't need to update anything.

I tested this by going into `chrome://flags` and setting these flags:



The website seems to function as normal, and the warning messages are not displayed in console. Might be worth someone checking on another environment other than chrome if possible?

**Comment by emab** [comment]
If someone else can test what I did above and confirm that our site will not be affected by the warnings we can close this issue.
